types/office-js-preview/index.d.ts

```diff
@@ -573,6 +573,59 @@ declare namespace Office {
         onVisibilityModeChanged(
             listener: (message: VisibilityModeChangedMessage) => void,
         ): Promise<() => Promise<void>>;
+
+        /**
+         * Represents a notification dialog that can appear when the user closes a document, before allowing the document to close.
```

Comment:
```suggestion
         * Represents a modal notification dialog that can appear when the user attempts to close a document. The document will  not close until the user responds..
```

Reply:
I kept getting errors from GitHub when trying to commit your suggestions directly, so I added them in a new commit.

Reply:
Yeah. You can't accept suggestions on DefinitelyTyped for some reason. I only use them as an easy way of describing what I think the change should be.